pkg/extension/smartagentextension/go.mod

```diff
@@ -7,15 +7,14 @@ require (
 	github.com/signalfx/signalfx-agent v1.0.1-0.20230104182534-9eee411fe305
 	github.com/signalfx/splunk-otel-collector/tests v0.83.0
 	github.com/stretchr/testify v1.8.4
-	go.opentelemetry.io/collector/component v0.93.0
-	go.opentelemetry.io/collector/confmap v0.93.0
-	go.opentelemetry.io/collector/extension v0.93.0
+	go.opentelemetry.io/collector/component v0.94.1
+	go.opentelemetry.io/collector/confmap v0.94.1
+	go.opentelemetry.io/collector/extension v0.94.1
```

Comment:
0.94.1 used here

Reply:
Isn't it correct that we want `v0.94.1` of `core` and `v0.94.0` of `contrib`? Or am I missing something?